src/main/java/org/jenkins/plugins/lockableresources/LockableResource.java

```diff
@@ -172,18 +187,21 @@ public boolean isEphemeral() {
 
   @Exported
   public String getLabels() {
-    return String.join(" ", this.labels);
+    if (this.labelsAsList == null) {
+      return "";
+    }
+    return String.join(" ", this.labelsAsList);
```

Comment:
Wouldn't it make more sense to remove the old thing completely?

Reply:
Yes it would be better.  But it is public and exported. I will try to remove the usage in the plugin and mark the function as deprecated.